modules/cudl-data-processing/s3.tf

```diff
@@ -10,7 +10,7 @@ resource "aws_s3_bucket" "transcriptions-bucket" {
   bucket = lower("${var.environment}-${var.transcriptions-bucket-name}")
 }
 
-
+# NOTE: is this needed for the transcription bucket?
```

Comment:
We're going to be removing the transcription bucket, and this config with it.   I have a separate pull request including that so no action needed.